frontend/insurance/src/app/modules/edit-claim/edit-files/edit-files.component.html

```diff
@@ -1,81 +1,93 @@
-<ly-card>
-  <ly-card-content>
-    <div class="mrl mll">
-      <ly-grid container spacing="16 8@XSmall">
-        <ly-grid item col="12 12@XSmall">
-          <h1 lyTyp="headline" gutterBottom>Files On Claim</h1>
+<div style="height:100%">
+  <ly-card>
+    <ly-card-content>
+      <div class="mrl mll">
+        <ly-grid container spacing="16 8@XSmall">
+          <ly-grid item col="12">
+            <h1 lyTyp="headline" gutterBottom>Files On Claim</h1>
```

Comment:
This shouldn't be capitalized if the Vehicle Information tab's header isn't capitalized.